spec/test_apps/app/dumpers/response_dumper_config.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+ResponseDumper.configure do |config|
+  config.before_all do
```

Comment:
IMO, we should match the rspec API:

```suggestion
  config.before(:all) do
```

Reply:
We do have `before` and like rspec, it should default to `before(:each)` -- as in each dump.

https://github.com/Pioneer-Valley-Books/rails-response-dumper/blob/7ca6483b4dfa359f3f9f9d8fa42ed6e4fc66adf5/lib/rails_response_dumper/defined.rb#L34-L36